index.html

```diff
@@ -0,0 +1,695 @@
+                        <iframe src="https://carbon.now.sh/embed/VBdyRItRjCzcDIczEqCw"
+                            style="width: 100%; height: 294px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+
+
+
+            </div>
+            <!-- ALERTS -->
+            <div class="component" id="alerts-id">
+                <h1>Alert</h1>
+                <p class="lead">Alerts are push notifications</p>
+
+                <div class="alert alert-primary">
+                    I am Primary alert
+                </div>
+                <div class="alert alert-secondary">
+                    I am Secondary alert
+                </div>
+                <div class="alert alert-info">
+                    I am Info alert
+                </div>
+                <div class="alert alert-warning">
+                    I am Warning Alert
+                </div>
+                <div class="alert alert-success">
+                    I am Success alert
+                </div>
+                <div class="alert alert-danger">
+                    I am Danger alert
+                </div>
+
+                <div class="display-none">
+
+                    <div onclick="showCode('alerts')" class="pre-div">
+                        <i class="fas fa-angle-double-right"></i>
+                    </div>
+
+                    <div class="pre-tag" id='alerts' style="display: none;">
+                        <iframe src="https://carbon.now.sh/embed/kE2CZFsm0MoxHnCcunHx"
+                            style="width: 100%; height: 510px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+
+
+            </div>
+            <!-- BADGES -->
+            <div class="component" id="badges-id">
+                <h1>Badges</h1>
+                <p class="lead">Badges in Web Design usually helps to highlight certain attributes of an item.</p>
+
+                <div class="category">
+                    <button class="btn btn-google round badge-relative">
+                        <i class="fab fa-google"></i>
+                        <span class="badge">3</span>
+                    </button>
+
+                    <button class="btn btn-secondary badge-relative">
+                        Notifications<span class="badge">1</span>
+                    </button>
+                </div>
+
+                <div class="category">
+                    <div class="badge-div">
+                        <img src="/assets/Images/avatar1.jpg" class="avatar avatar-lg" />
+                        <span class="badge-image left"></span>
+                    </div>
+
+                    <div class="badge-div">
```

Comment:
badge-div class does not sound generalised, can you please make it to badge-container or something else?

Reply:
done